packages/zipkin-transport-kafka/src/KafkaLogger.js

```diff
@@ -1,5 +1,5 @@
 const kafka = require('kafka-node');
-const {serializeSpan} = require('zipkin');
+const serializeSpan = require('../../../internals/serialize-span');
```

Comment:
how does the "internals" approach for those using npm? Is there a specific project setup they would need?

Reply:
I was under the assumtion it would be inlined, but it isn't. I will do some research how to do it or if it's impossible I will just create a symlink 👍  [preval](https://github.com/kentcdodds/babel-plugin-preval) might be an option for inlining 🤔

Reply:
Symlinks don't work on Windows